packages/web-components/src/components/va-button/va-button.tsx

```diff
@@ -124,6 +126,32 @@ export class VaButton {
     return this.text;
   };
 
+  private handleSubmit() {
+    // const type = this.el.shadowRoot.querySelector('button')?.type;
+    // console.log(this.el.shadowRoot.querySelector('button'), 'XX');
+    // if (type !== 'submit') {
+    //   return;
+    // }
+    if (this.submit === undefined) {
+      return;
+    }
+    const theForm = this.el.closest('form');
```

Comment:
You should mention in the `submit` prop description that it requires a `form` element.